pkg/plugins/models.go

```diff
@@ -310,32 +311,34 @@ type DataSourceDTO struct {
 }
 
 type PanelDTO struct {
-	ID              string          `json:"id"`
-	Name            string          `json:"name"`
-	AliasIDs        []string        `json:"aliasIds,omitempty"`
-	Info            Info            `json:"info"`
-	HideFromList    bool            `json:"hideFromList"`
-	Sort            int             `json:"sort"`
-	SkipDataQuery   bool            `json:"skipDataQuery"`
-	ReleaseState    string          `json:"state"`
-	BaseURL         string          `json:"baseUrl"`
-	Signature       string          `json:"signature"`
-	Module          string          `json:"module"`
-	Angular         AngularMeta     `json:"angular"`
-	LoadingStrategy LoadingStrategy `json:"loadingStrategy"`
-	ModuleHash      string          `json:"moduleHash,omitempty"`
+	ID              string            `json:"id"`
+	Name            string            `json:"name"`
+	AliasIDs        []string          `json:"aliasIds,omitempty"`
+	Info            Info              `json:"info"`
+	HideFromList    bool              `json:"hideFromList"`
+	Sort            int               `json:"sort"`
+	SkipDataQuery   bool              `json:"skipDataQuery"`
+	ReleaseState    string            `json:"state"`
+	BaseURL         string            `json:"baseUrl"`
+	Signature       string            `json:"signature"`
+	Module          string            `json:"module"`
+	Angular         AngularMeta       `json:"angular"`
+	LoadingStrategy LoadingStrategy   `json:"loadingStrategy"`
+	ModuleHash      string            `json:"moduleHash,omitempty"`
+	Locales         map[string]string `json:"locales,omitempty"`
```

Comment:
I think this should be called `Languages` or `Translations` (presuming its a map of language code to translation file?

'Locale' is what we call how values are formatted.

https://github.com/grafana/grafana/blob/e5352a76cf3ff2a4de37abd735a514d8c91b303c/pkg/api/index.go#L50-L54

Reply:
Great suggestion, I'll change this 👍

Reply:
Just to be super clear with this, the folder `https://github.com/grafana/grafana/tree/main/public/locales` should be renamed `languages` if we want to have to "same" semantics everywhere. Is that a correct assumption @joshhunt ?  I'm asking because this PR assumes that all language files for plugins exist under `src/locales` https://github.com/grafana/grafana/blob/hugoh/i18n-locales/pkg/plugins/manager/loader/assetpath/assetpath.go#L158 (following the example from core grafana). Do we want to keep it like this as the one exception of the rule sort of thing 😉

Reply:
Great, I'll do that 👍

Reply:
Fixed 👍